<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigEditor</class>
 <widget class="QWidget" name="ConfigEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>253</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>357</width>
    <height>253</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>357</width>
    <height>253</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration Editor</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>341</width>
     <height>234</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_Name">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Name"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_World">
          <property name="text">
           <string>World</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_World"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_World">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_MaxPlayers">
            <property name="text">
             <string>Max Players (1-255)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_MaxPlayers">
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>8</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_Port">
            <property name="text">
             <string>Port (1024-65535)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_Port">
            <property name="minimum">
             <number>1024</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>7777</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_Password">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Password"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_MOTD">
          <property name="text">
           <string>MOTD</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_MOTD"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_AutoCreateName">
          <property name="text">
           <string>Auto Create Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_AutoCreateName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_AutoCreateSize">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>Auto Create Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_AutoCreateSize">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>Small</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Medium</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Large</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_Language">
            <property name="maximumSize">
             <size>
              <width>47</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Language</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_Language">
            <property name="maximumSize">
             <size>
              <width>63</width>
              <height>20</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>English</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>German</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Italian</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>French</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Spanish</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_Secure">
            <property name="maximumSize">
             <size>
              <width>33</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Secure</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_Secure">
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QPushButton" name="pushButton_Save">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Cancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
